checker/exported_ident_doc_checker.go

```diff
@@ -3,19 +3,39 @@ package checker
 import (
 	"fmt"
 	"go/ast"
+	"strings"
+
+	"github.com/uber-go/mapdecode"
 )
 
 func init() {
 	must(Register("exported_ident_doc", NewExportedIdentDocChecker))
 }
 
+// ExportedIdentDocCheckerConfig describes the configuration of a ExportedIdentDocChecker.
+type ExportedIdentDocCheckerConfig struct {
+
+	// CheckDocPrefix signals if the checker should ensure that every doc comment begins
+	// with the name of the item it describes.
+	CheckDocPrefix bool `mapdecode:"check_doc_prefix"`
```

Comment:
Can you use `yaml` instead of `mapdecode`? `yaml` is used in file matchers for similar purpose: https://github.com/s2gatev/lingo/blob/master/file/glob_matcher.go#L16